django_ibmi/operations.py

```diff
@@ -229,14 +209,12 @@ def field_cast_sql(self, db_type, internal_type=None):
         else:
             return " %s"
 
-    def fulltext_search_sql(self, field_name):
-        sql = "WHERE %s = ?" % field_name
-        return sql
-
     # Function to return value of auto-generated field of last executed
     # insert query.
     def last_insert_id(self, cursor, table_name, pk_name):
-        return cursor.last_identity_val
+        operation = 'SELECT IDENTITY_VAL_LOCAL() FROM SYSIBM.SYSDUMMY1'
+        result = cursor.execute(operation)
+        return result.fetchone()[0]
```

Comment:
```suggestion
        return result.fetchval()
```

PyODBC nicety: https://github.com/mkleehammer/pyodbc/wiki/Features-beyond-the-DB-API#fetchval